internal/api/v1/stage.go

```diff
@@ -207,7 +208,7 @@ func createCertificate(ctx context.Context, cfg *rest.Config, app gitea.App, sys
 				"kind" : "ClusterIssuer"
 			}
 		}
-        }`, app.Name, app.Org, app.Name, systemDomain, app.Name, app.Name, systemDomain, issuer)
+        }`, app.Name, app.Org, names.TruncateMD5(fmt.Sprintf("%s.%s", app.Name, systemDomain), 64), app.Name, app.Name, systemDomain, issuer)
```

Comment:
Can we put a comment (in code) on why we do this here? Something about the character limit in the CN field which is not a problem in the dnsNames. Maybe also mention that generally the CN field is deprecated and the SANs are preferred.

Reply:
Will do.

Reply:
Done with commit 4a778220aa55258c8507c6fe61944dfcf63632b8

Reply:
And rebased due to conflict with new tekton staging work.